### PR TITLE
Move Component PHPUnit tests

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Based on #733

### CODE REVIEW

**Review by Prometee — APPROVED** [review]

**Comment by loic425** [comment]
@Prometee I've rebased this PR, could you review again please?

**Review by oallain — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -108,6 +108,7 @@
             "Sylius\\Bundle\\ResourceBundle\\spec\\": "src/Bundle/spec/",
             "Sylius\\Bundle\\ResourceBundle\\Tests\\": "tests/Bundle/",
             "Sylius\\Component\\Resource\\spec\\": "src/Component/spec/",
+            "Sylius\\Component\\Resource\\Tests\\": "tests/Component/",
```
Comment (jakubtobiasz) [inline composer.json:111]:
```suggestion
            "Tests\\Sylius\\Bundle\\ResourceBundle\\": "tests/Bundle/",
            "Sylius\\Component\\Resource\\spec\\": "src/Component/spec/",
            "Tests\\Sylius\\Component\\Resource\\": "tests/Component/",
```
Wdyt? That's how we usually create namespaces for `tests` dir. It doesn't suggest the tests are part of the `Sylius\Component\Resource` namespace.

Comment (loic425) [reply composer.json:111]: Hum, yeah, you're right, but that could be harder to review for now and resource bundle tests are with the same namespace.
Could we merge like that and making another PR with this change for component and bundles?

**Comment by loic425** [comment]
Hum I have another Idea...

**Pull request closed without merging**